### PR TITLE
C++11 type aliases should be treated similar to typedefs

### PR DESCRIPTION
**Pull request by bolshakov-a**
Seemingly useless line of code removed by the way.

### CODE REVIEW

File: iwyu.cc [hunk 1]
```diff
@@ -3941,10 +3941,9 @@ class IwyuAstConsumer
   // TODO(csilvers): we can probably relax this rule in .cc files.
   // TODO(csilvers): this should really move into IwyuBaseASTVisitor
   // (that way we'll correctly identify need for hash<> in hash_set).
-  // This is a Traverse*() because Visit*() can't call HandleFunctionCall().
-  bool TraverseTypedefDecl(clang::TypedefDecl* decl) {
-    if (!Base::TraverseTypedefDecl(decl))
-      return false;
+  // This is called from Traverse*() because Visit*()
+  // can't call HandleFunctionCall().
+  bool CheckAliasedClassMethods(TypedefNameDecl* decl) {
```
Comment (kimgr) [inline iwyu.cc:3946]: I'd prefer `Handle...` to `Check...` here. "Check" implies "try-and-fail-hard" in IWYU, given the `CHECK_` macros.

**Review by kimgr — COMMENTED** [review]
Looks great! One naming nit, otherwise lgtm.

File: iwyu.cc (hunk 1 above)
Comment (bolshakov-a) [reply iwyu.cc:3946]: Thanks, fixed.

Comment (bolshakov-a) [reply iwyu.cc:3946]: I`m sorry, actually, isn't.

Comment (bolshakov-a) [reply iwyu.cc]: Fixed at now.

**Comment by kimgr** [comment]
Wonderful, thank you!

**Pull request merged**